lib/next_rails/bundle_report.rb

```diff
@@ -49,7 +53,16 @@ def compatibility(rails_version: nil, include_rails_gems: nil)
 <%= incompatible_gems.length.to_s.red %> gems incompatible with Rails <%= rails_version %>
       ERB
 
-      puts ERB.new(template, trim_mode: "-").result(binding)
+      erb_version = ERB.version
+      if erb_version =~ /erb.rb \[([\d\.]+) .*\]/
```

Comment:
in older erb versions, the `version` string was something like `erb [2.2.0 2019-10-04]` instead of just the version, but it changed in newer versions to just `4.0.0`